WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/Tap to Pay Education/TapToPayEducationStepViewModel.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+
+final class TapToPayEducationStepViewModel: ObservableObject {
+    let title: String
+    let imageName: String
+    let descriptionSteps: [String]
+
+    private lazy var numberFormatter: NumberFormatter = {
+        let numberFormatter = NumberFormatter()
+        numberFormatter.numberStyle = .decimal
+        numberFormatter.locale = Locale.current
+        return numberFormatter
+    }()
```

Comment:
Could you use the SwiftUI's `Text(_:, format: .number)` view init instead?

Reply:
Thanks for the suggestion. Yes, I tested and this Text with a number is localized automatically as well.